packages/ra-ui-materialui/src/list/filter/FilterButton.stories.tsx

```diff
@@ -372,3 +383,65 @@ export const WithAutoCompleteArrayInput = (args: {
         </MemoryRouter>
     );
 };
+
+export const WithButtonVariant = () => {
+    const postFilters: React.ReactElement[] = [
+        <TextInput
+            label="Title"
+            source="title"
+            defaultValue="Accusantium qui nihil voluptatum quia voluptas maxime ab similique"
+        />,
+    ];
+    return (
+        <MemoryRouter>
+            <Admin
+                dataProvider={fakerestDataProvider(data)}
+                store={memoryStore()}
+            >
+                <Resource
+                    name="posts"
+                    list={
+                        <PostList
+                            postFilters={postFilters}
+                            args={{}}
+                            buttonProps={{
+                                variant: 'outlined',
+                            }}
+                        />
+                    }
+                />
+            </Admin>
+        </MemoryRouter>
+    );
+};
+
+export const WithButtonSize = () => {
```

Comment:
<!-- Polish comments are like a suggestion, where there is nothing necessarily wrong with the relevant content, there's just some ways to immediately improve the quality.  -->
**polish:** ‏May we name it as the prop name

```suggestion
export const Size = () => {
```